src/globals/settings/_typography.scss

```diff
@@ -205,7 +205,7 @@ hr {
   &.nhsuk-h-line {
     border: 0;
     border-top: 1px solid $nhsuk-border-colour;
-    margin: 48px 0 0;
+    margin: 40px 0 0;
```

Comment:
Need to check with @davidhunter08 that this change is OK as its a change to the global frontend library.